### ISSUE TITLE
VirusTotal.com: One engine detected this file. (False positive?)

### ISSUE THREAD

**Issue by Soninmyeyes** [OldPlanets/Wireboard#10]
On VirusTotal.com, one engine flags wireboard-setup.exe:

Engine: Bkav
Warning: HW32.Packed.

From a few forum posts, it sounds like this is probably a false positive, roughly translatable as, "this file is packaged in such a way that I can't scan it properly."

Since no other engines flag the file, I'm inclined to go with the false-positive interpretation in this case; however, it might be worth addressing.

**Comment by OldPlanets** [comment]
Yes, its a false positive. I don't know how I could address it, the file isn't packed in any special way it's just normally compiled. If you want to be super safe, you can always compile the code yourself from this github of course.
